src/subdomains/generic/kyc/dto/update-kyc-log.dto.ts

```diff
@@ -0,0 +1,12 @@
+import { IsEnum, IsNotEmpty, IsString } from 'class-validator';
+import { ManualRiskRate } from '../entities/kyc-log.entity';
+
+export class UpdateKycLogDto {
+  @IsNotEmpty()
+  @IsEnum(ManualRiskRate)
+  manualRiskRate: ManualRiskRate;
+
+  @IsNotEmpty()
+  @IsString()
+  comment: string;
+}
```

Comment:
If `manualRiskRate` is required we probably should rename the endpoint and DTO to setManualRisk? 😉